examples/ekfutils/vins_one.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from pymlg import SE3
+
+import examples.ekfutils.common as common
+import miluv.utils as utils
+
+# EKF parameters
+state_dimension = 6
+np.random.seed(0)
+
+# Covariance matrices
+P0 = np.diag([0.01, 0.01, 0.01, 0.1, 0.1, 0.1]) # Initial state covariance
+
+imu_noise_params = utils.get_imu_noise_params("ifo001", "px4")
+Q = np.diag([
+    imu_noise_params["gyro"][0]**2, 
+    imu_noise_params["gyro"][1]**2,
+    imu_noise_params["gyro"][2]**2,
+    0.1 , 0.1, 0.1
+]) # Process noise covariance
+
+R_range = 0.3**2 # Range measurement noise covariance
+R_height = 0.5**2 # Height measurement noise covariance
+
+class EKF:
```

Comment:
This is picky so feel free to ignore it if it's too much work to address:

This EKF class has the class variables P, anchors, and tag_moment_arms. However, these variables don't describe the filter itself, but are just variables that are used in the EKF methods. I understand that they're in here for convenience, but they aren't properties of the filter, but of the robot and the environmental set up.

I think that they should either be moved out of the EKF class or the EKF class can be renamed to EKF_Problem, or something similar. That being said, nothing here strictly needs changing, so if you don't feel like doing it, that's fine with me.

Reply:
I didn't want to define too many classes and abstractions to keep this as easy to read as possible. I get that for reusability and whatnot there is a lot of value in separating an EKF into different classes, but for readability I decided to put it all in one EKF and just have it named simply as EKF

Reply:
lmk what you think